nms/app/packages/magmalte/app/views/traffic/PolicyFlows.js

```diff
@@ -335,41 +359,34 @@ export default function PolicyFlowsEdit(props: Props) {
     props.onChange({...props.policyRule, flow_list: flowList});
   };
 
+  const flowList = props.policyRule.flow_list || [];
   return (
-    <>
-      <DialogContent
-        data-testid="networkInfoEdit"
-        className={props.dialogClass}>
-        <List>
-          <Typography
-            variant="caption"
-            display="block"
-            className={props.descriptionClass}
-            gutterBottom>
-            {"A policy's flows determines how it routes traffic"}
-          </Typography>
-          <ListItem dense disableGutters />
-          {props.policyRule.flow_list.length > 0 && (
-            <ListItem disableGutters>
-              <Typography variant="h6">Flows</Typography>
-            </ListItem>
-          )}
-          {(props.policyRule.flow_list || []).slice(0, 30).map((flow, i) => (
-            <ListItem key={i} disableGutters>
-              <PolicyFlowFields2
-                index={i}
-                flow={flow}
-                handleDelete={handleDeleteFlow}
-                onChange={onFlowChange}
-              />
-            </ListItem>
-          ))}
-          Add New Flow
-          <IconButton onClick={handleAddFlow}>
-            <AddCircleOutline />
-          </IconButton>
-        </List>
-      </DialogContent>
-    </>
+    <div data-testid="flowEdit">
+      <Text weight="medium" variant="subtitle2" className={classes.description}>
+        {"A policy's flows determines how it routes traffic"}
```

Comment:
Taking a look at this again, I think we need some better docs. Created an issue 3290.

#3290

Reply:
We have to add policy configuration section. https://magma.github.io/magma/docs/nms/nms_traffic.
We can update this document